mapfishapp/src/main/webapp/app/addons/locateme/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+    "js": [
+        "js/main.js"
+    ],
+    "css": [
+        "css/main.css"
+    ],
+    "default_options": {
+        "target": "bbar_7",
```

Comment:
remove target in here (or user cannot choose to have the addon in the tools menu)